apps/feedback/services/feedback.go

```diff
@@ -133,16 +134,21 @@ func (service FeedbackService) getFeedbackDetail(feedback *feedbackSerializers.F
 					FeedbackFormContentID: feedBackFormContent.ID,
 				}).
 				FirstOrCreate(&questionResponse)
-
+			questionOptions := utils.ByteToMap(question.Options)
```

Comment:
as discussed let's create model/serializer helper methods where ever we see it's apt

Reply:
Done

Reply:
Done